packages/Datepicker/src/Datepicker.tsx

```diff
@@ -21,7 +21,7 @@ import Calendar from "./components/Calendar";
 
 import "./datepicker.scss";
 
-interface Date { utc: string; local: string }
+interface Date { utc: string; local: string | null }
```

Comment:
local: string | null

is that related to the provider or a package update?

Reply:
It's most likely related to a package update. It complained without it.